kyuubi-server/src/test/resources/flink_conf/zoo.cfg

```diff
@@ -0,0 +1,36 @@
+################################################################################
```

Comment:
Can zookeeper peer be skipped?

Reply:
Copied from `$FLINK_HOME/conf`

Reply:
why?

Reply:
I can not find the way to overwrite the configure defined in `$FLINK_HOME/conf/flink-conf.yaml`, thus copied the whole `$FLINK_HOME/conf` to `flink_conf`, and then modify the `flink_conf/flink-conf.yaml`, `zoo.cfg` is not required for our test, because we don't turn on HA.

The current approach is copy ``$FLINK_HOME/conf` with minimal modification and keep others as-is, or we just pick the required conf files?

Reply:
I don't follow your thoughts, 'when you need to modify one config, you cp the whole dir.`

Reply:
1. we can only use `FLINK_CONF_DIR`, there is no option like `--conf-file=xxx.yaml`
2. I can not found a way to overwrite a property when `bin/start-cluster`

any suggestion?